### PR TITLE
wallpaper_community: Add recipe

### PR DESCRIPTION
**Pull request by win8linux**
This package contains community-made wallpapers from https://github.com/haiku/artwork/pull/9, mostly from the initial submissions of the [Call for Wallpapers](https://www.haiku-os.org/news/2020-10-04-wallpaper-contest/).

Need advice regarding what license to put in the recipe, as the wallpapers are licensed with a mix of CC-BY and CC-BY-SA 4.0-International. `Public Domain` is just a placeholder for now.

### CODE REVIEW

**Comment by waddlesplash** [comment]
You can put more than one license in a recipe by having line breaks between them. Other recipes do this already.

File: haiku-data/wallpaper_community/wallpaper_community-0.1.recipe [hunk 1]
```diff
@@ -2,7 +2,10 @@ SUMMARY="Wallpapers from the Haiku community"
 DESCRIPTION="Wallpapers created by members of the Haiku community"
 HOMEPAGE="https://github.org/haiku/artwork"
 COPYRIGHT="Various Authors"
-LICENSE="Public Domain"
+LICENSE="
+	CC-BY 4.0
+	CC-BY-SA 4.0
+	"
```
Comment (humdingerb) [inline haiku-data/wallpaper_community/wallpaper_community-0.1.recipe:8]: I think the endquotes belong to the line above.

File: haiku-data/wallpaper_community/wallpaper_community-0.1.recipe [hunk 2]
```diff
@@ -2,7 +2,10 @@ SUMMARY="Wallpapers from the Haiku community"
 DESCRIPTION="Wallpapers created by members of the Haiku community"
 HOMEPAGE="https://github.org/haiku/artwork"
```
Comment (humdingerb) [inline haiku-data/wallpaper_community/wallpaper_community-0.1.recipe:3]: I wonder if using https://www.haiku-os.org/news/2020-10-04-wallpaper-contest/ would be better?

File: haiku-data/wallpaper_community/wallpaper_community-0.1.recipe [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+PROVIDES="
+	wallpaper_community = $portVersion
+	"
+REQUIRES=""
+
+BUILD_REQUIRES=""
+BUILD_PREREQUIRES="
+	cmd:cp
+	cmd:rm"
```
Comment (humdingerb) [inline haiku-data/wallpaper_community/wallpaper_community-0.1.recipe:26]: For some reason everywhere else but the DESCRIPTION, multi-line end quotes are put in the next empty line with a tab indention. I'm just remarking how the other recipes do it, I'm a bit embarassed to point out these tiny details...

Comment (win8linux) [reply haiku-data/wallpaper_community/wallpaper_community-0.1.recipe:26]: Honestly, formatting can be somewhat inconsistent across recipes.

**Comment by humdingerb** [comment]
Thanks you worked on that!

**Pull request merged**